CMakeSettings.json

```diff
@@ -0,0 +1,55 @@
+{
+  // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
```

Comment:
https://stackoverflow.com/questions/244777/can-comments-be-used-in-json#244858
tl;dr //comments are a violation of spec, suggested solutions are a "_comments" field or parsing through JSMin before use